packages/ui-react/src/components/Shelf/Shelf.tsx

```diff
@@ -46,7 +47,7 @@ export type ShelfProps = {
   accessModel: AccessModel;
   isLoggedIn: boolean;
   hasSubscription: boolean;
-  posterAspect?: string;
+  posterAspect?: PosterAspectRatio;
```

Comment:
I think we removed the type a few months ago. Is it really needed?

There is no guarantee that we're working with a valid AspectRatio from the custom params.

Reply:
Then that would break our scss class, which depends on these ratio's (and caused the type error that lead me to refactor this).